### PR TITLE
[Do not merge] Fix/gripper interface

### PR DESCRIPTION
**Pull request by LarsJanssenTUe**
Merge after tue-robotics/hero_bridge#12

### CODE REVIEW

File: robot_skills/src/robot_skills/arms.py [hunk 1]
```diff
@@ -646,7 +646,7 @@ def send_gripper_goal(self, state, timeout=5.0, max_torque=0.1):
             goal.command.direction = GripperCommand.OPEN
         elif state == GripperState.CLOSE:
             goal.command.direction = GripperCommand.CLOSE
-            goal.command.max_torque = max_torque
+            goal.command.max_force = max_force
```
Comment (MatthijsBurgh) [inline robot_skills/src/robot_skills/arms.py]: This is a msg, so change that too.

Comment (LarsJanssenTUe) [reply robot_skills/src/robot_skills/arms.py]: it seems I indeed missed a few things.... not even only the msg.... Sorry for that! Good catch!

**Comment by LarsJanssenTUe** [comment]
Please do not merge yet! I am still in contact with Toyota about this issue!

**Comment by PetervDooren** [comment]
Ping @LarsJanssenTUe, Is this PR something we should still work on sometime? Or shall we close it?

**Pull request closed without merging**